test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -21,33 +21,41 @@
 
 'use strict';
 const common = require('../common');
+const { execSync } = require('child_process');
+
+const sysctlOutput = execSync('sysctl net.ipv4.ip_unprivileged_port_start').toString();
```

Comment:
Wrap this in a `if (common.isLinux) { /* ... */ }` and call `common.skip()` when 42 is unprivileged.